Blish HUD/Blish HUD.csproj

```diff
@@ -1,766 +1,226 @@
-    <EmbeddedResource Include="Strings\GameServices\SettingsService.es.resx" />
-    <EmbeddedResource Include="Strings\GameServices\SettingsService.fr.resx" />
-    <EmbeddedResource Include="Strings\GameServices\SettingsService.resx">
+    <EmbeddedResource Update="Strings\GameServices\SettingsService.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>SettingsService.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(MSBuildExtensionsPath)\MonoGame\v3.0\MonoGame.Content.Builder.targets" />
-  <PropertyGroup>
-    <PostBuildEvent>rd /s /q "lv", "ms-MY", "mt", "nb", "nb-NO", "nl", "pl", "pt", "ro", "ru", "sk", "sl", "sr", "sr-Latn", "sv", "tr", "uk", "uz-Cyrl-UZ", "uz-Latn-UZ", "vi", "zh-CN"</PostBuildEvent>
-  </PropertyGroup>
-  <PropertyGroup>
-    <PreBuildEvent>powershell.exe -ExecutionPolicy Bypass -NoProfile -NonInteractive -File "$(ProjectDir)BuildScripts\RefBuilder.ps1" -Project "$(ProjectDir)\" -Output "$(OutDir)\"</PreBuildEvent>
-  </PropertyGroup>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
+
+  <ItemGroup>
+    <PackageReference Include="Adhesive" Version="2.0.1-alpha" PrivateAssets="all" />
+    <PackageReference Include="AsyncClipboardService" Version="1.7.1" PrivateAssets="all" />
+    <PackageReference Include="Costura.Fody" Version="4.1.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="DynamicLanguageRuntime" Version="1.2.2" PrivateAssets="all" />
+    <PackageReference Include="EntryPoint" Version="1.3.0" PrivateAssets="all" />
+    <PackageReference Include="Flurl.Http" Version="2.4.2" PrivateAssets="all" />
+    <PackageReference Include="Fody" Version="6.1.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Gw2Sharp" Version="[0.9.6]" />
+    <PackageReference Include="Humanizer.Core.de" Version="2.6.2" PrivateAssets="all" />
+    <PackageReference Include="Humanizer.Core.es" Version="2.6.2" PrivateAssets="all" />
+    <PackageReference Include="Humanizer.Core.fr" Version="2.6.2" PrivateAssets="all" />
+    <PackageReference Include="JsonSubTypes" Version="1.6.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
+    <PackageReference Include="MonoGame.Content.Builder" Version="3.7.0.9" PrivateAssets="all" />
+    <PackageReference Include="MonoGame.Extended" Version="3.7.0" PrivateAssets="all" />
```

Comment:
Could we get the following included with modules as well?
- MonoGame.Extended (but not MonoGame.Extended.Content.Pipeline)
- Fody & Costura (assuming we don't go with ILMerge for modules)
- AsyncClipboardService

Reply:
Done :)